### DISCUSSION TITLE
Integrating OTBR with Rainmaker

### DISCUSSION THREAD

**Discussion by YasithRodrigo** [openthread/openthread#10039] [Q&A]
Hello, 
I am reaching out to seek assistance regarding a technical challenge I'm currently facing. Specifically, I am in the process of integrating OTBR (OpenThread Border Router) with Rainmaker, but I have encountered difficulties due to the lack of available examples or documentation. Can you provide a sample code? Any support is highly appreciated.

**Comment by wgtdkp** [comment]
QQ: what is "Rainmaker"?

**Comment by chshu** [comment]
@YasithRodrigo Are you looking for the Espressif Rainmaker over Thread solution? https://rainmaker.espressif.com/

If yes, could you please file an issue to the rainmaker repo: https://github.com/espressif/esp-rainmaker? Basically, as long as the Thread BR supports NAT64, no other Rainmaker specific thing required.
